app/views/guests/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Guest Name: <%= @guest.name %></h1>
+<% @guest.rooms.each do |room| %>
+  <h3>Room Number(s): </h3><%= room.id %>
+  <h3>Room Rate: </h3><%= room.rate %>
+  <h3>Room Suite: </h3><%= room.suite %>
+  <h3>Hotel Name: </h3><%=  room.hotel.name %>
```

Comment:
You use labels here which is good, you'll just want to start bringing those labels into your tests.